packages/web-components/examples/codesandbox/components/content-block-headlines/src/index.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import '@carbon/ibmdotcom-web-components/es/components/content-item/content-item-copy';
+import '@carbon/ibmdotcom-web-components/es/components/content-block-headlines/content-block-headlines';
+import '@carbon/ibmdotcom-web-components/es/components/content-block-headlines/content-block-headlines-item';
```

Comment:
Some modern module bundlers require `.js` extension in the `import`:

```suggestion
import '@carbon/ibmdotcom-web-components/es/components/content-item/content-item-copy.js';
import '@carbon/ibmdotcom-web-components/es/components/content-block-headlines/content-block-headlines.js';
import '@carbon/ibmdotcom-web-components/es/components/content-block-headlines/content-block-headlines-item.js';
```